app/assets/stylesheets/navbar.scss

```diff
@@ -28,4 +28,23 @@
   .navbar-collapse {
     border-top: 0;
   }
+
+  @media and (min-width: 768px) and (max-width: 991px){
```

Comment:
The first "and" after media needs to be removed. Once that's removed, this looks great and is ready to go! Can you remove that and check in the fix so we can pull this change in right away? So sorry for the long delay! We are working to clean up all existing pull requests as soon as possible :)